api/transport/router.go

```diff
@@ -40,7 +40,8 @@ type Procedure struct {
 	// HandlerSpec specifiying which handler and rpc type.
 	HandlerSpec HandlerSpec
 
-	// Encoding of the handler, for introspection.
+	// Encoding of the handler, optional, used for introspection, and used for
+	// routing if present.
```

Comment:
More sentence, less list please.

> Encoding of the handler. This field is optional.

We don't need to mention exactly what it's used for because that list can expand (as it has already in this change).